src/controller/account.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const account = express.Router();
+const { User } = require('../models');
+
+account.get('/user', async (req, res) => {
+  try {
+    const { _id } = req.user;
+    const user = await User.findById(_id, '-watchlists -investments');
+    return res.send(user);
```

Comment:
The same as in the investment endpoint. Are you sure about the response format? Aren't there any unnecessary fields?